src/com/javabasic/lesson9/ZeroSort.java

```diff
@@ -0,0 +1,17 @@
+package com.javabasic.lesson9;
+
+import java.util.Arrays;
+
+public class ZeroSort {
+
+    static int[] arrayWithZero = {2, 1, 0, 5, 0, 9};
+
+    public static void main(String[] args) {
+
+        Arrays.sort(arrayWithZero);
```

Comment:
it's ok to use the sort method, but you need to implement your own realization at least once)